Components/Pages/About/Component/about.html

```diff
@@ -0,0 +1,117 @@
+    <link rel="stylesheet" href="../Styles/about.css">
+
+    <title>About Page</title>
+</head>
+
+<body>
+    <header>
+        <nav class="navbar navbar-expand-md navbar-light bgp-light nav-change navbar-change-style">
+            <div class="container-fluid">
+                <a class="navbar-brand p-0 m-0" href="../../../../index.html"><img class="p-0 m-0 logo-container"
+                        src="../../Assets/icon-game.gif" alt="icon"></a>
+                <button class="navbar-toggler me-auto" type="button" data-bs-toggle="collapse"
+                    data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent"
+                    aria-expanded="false" aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+                <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                    <ul class="navbar-nav ms-auto mb-2 mb-lg-0">
+                        <li class="nav-item">
+                            <a class="nav-link active" aria-current="page" href="#">About</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link disabled">Program</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link disabled">Join</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link disabled">Sponsor</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link disabled">News</a>
+                        </li>
+                        <li class="nav-item mx-5 log-button">
+                            <a class="nav-link">CC Campain</a>
+                          </li>
+                    </ul>
+                </div>
+            </div>
+        </nav>
+    </header>
+    <div class="about-home container-fluid header-container">
+        <div class="container">
+            <div class="container text-center d-flex justify-content-center">
+                <h1 class="text-title">RANKING OF VIDEO GAMES IN 2021</h1>
+            </div>
+            <div class="container d-flex my-3 justify-content-center line-color"></div>
+
+            <div class="container p-5 text-center text-comment">
+                <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Quos voluptas natus, architecto
+                    laudantium nesciunt voluptates molestias, fugit atque accusantium placeat esse, exercitationem
+                    accusamus. Earum quidem soluta quis molestias autem sint! Lorem ipsum dolor sit amet consectetur
+                    adipisicing elit. Voluptatum asperiores culpa facilis aut esse, earum quibusdam expedita a ex
+                    vel</p>
+            </div>
```

Comment:
- [ ] The text in the about page is overflowing, causing a horizontal scroll that breaks the view. As stated before, please make sure not to use fixed widths to avoid this kind of problems.

![](https://i.imgur.com/i5r9IxI.gif)